pkg/maps/srv6map/policy.go

```diff
@@ -0,0 +1,184 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package srv6map
+
+import (
+	"fmt"
+	"net"
+	"unsafe"
+
+	"golang.org/x/sys/unix"
+
+	"github.com/cilium/cilium/pkg/ebpf"
+	"github.com/cilium/cilium/pkg/types"
+)
+
+const (
+	PolicyMapName4   = "cilium_srv6_policy_v4"
+	PolicyMapName6   = "cilium_srv6_policy_v6"
+	MaxPolicyEntries = 16384
```

Comment:
I think in the future we might want to make this (and other from your PR) configurable.